src/back/responses.ts

```diff
@@ -1780,8 +1780,14 @@ export function registerRequestCallbacks(state: BackState, init: () => Promise<v
   });
 
   state.socketServer.register(BackIn.UPLOAD_LOG, async (event) => {
+    // Filter entries
+    const username = os.userInfo().username;
+    const entries = state.log.filter(e => e !== undefined).map(e => {
+      e.content = e.content.replace(new RegExp(username, 'g'), '***');
```

Comment:
Try this:
```ts
const slashMatch = '([\/\\\\])'
e.content = e.content.replace(new RegExp(slashMatch + username + slashMatch, 'g'), '$1***$2');
```

Reply:
LGTM. Made the second slash optional for the rare case that the path ends with the user folder. Technically allows for a few fringe false positives, but I think that's alright.

Reply:
If the second slash is optional, then I'm pretty sure there's no reason to include it in the regex, and you could get an equivalent result by just matching `sepMatch + username` and replacing with `$1***`.

Reply:
Of course... because there's no more to the pattern after that. Woops XD.

Revised.